js/views/app_view.js

```diff
@@ -207,6 +207,12 @@
       dialog.once('devicePairingRequestAccepted', (pubKey, cb) =>
         Whisper.events.trigger('devicePairingRequestAccepted', pubKey, cb)
       );
+      dialog.on('devicePairingRequestRejected', pubKey =>
```

Comment:
any reason for `on` vs `once`?

Reply:
Multiple requests can queue up while the dialog is open, so there could be multiple Rejected events. As opposed to the Accepted event, which can occur only once.

Reply:
Fair enough 👍